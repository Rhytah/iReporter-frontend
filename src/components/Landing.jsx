import React, { Component } from 'react';
import '../assets/css/Landing.css';
import { Link } from 'react-router-dom';

class LandingPage extends Component {
  render() {
    return (
      <div>
        <div id="page-top">
          <nav id="navigation">
            <div className="container text-center">
              <ul className="navHold">
                <li>
                  <a href="/">Home</a>
                </li>

                <li>
                  <a href="/">Redflags</a>
                </li>
                <li>
                  <a href="/">Interventions</a>
                </li>
                <li className="subNav">
                  Get started
                  <ul>
                    <li>
                      <Link to="/signup">Sign up</Link>
                    </li>
                    <li>
                      <Link to="/login">Login</Link>
                    </li>
                  </ul>
                </li>
              </ul>
            </div>
            <div className="logosmall hidels">
              <img alt="pic" src="assets/img/testlogosmall.png" height="24px" />
            </div>
            <div className="navResponsive" />
          </nav>
          <div className="container">
            <article>
              <h1>Welcome to iReporter</h1>

              <p>
                Corruption is a huge bane to Africaâ€™s development. African countries must develop
                novel and localised solutions that will curb this menace, hence the birth of
                iReporter. iReporter enables any/every citizen to bring any form of corruption to
                the notice of appropriate authorities and the general public. Users can also report
                on things that needs government intervention
              </p>
            </article>
          </div>
        </div>
      </div>
    );
  }
}

export default LandingPage;
