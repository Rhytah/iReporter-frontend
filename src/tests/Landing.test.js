import React from 'react';
import { shallow } from 'enzyme';
import Landing from '../components/Landing';

describe('Landing Component', () => {
  it('matches the snapshot', () => {
    const component = shallow(<Landing />);
    expect(component).toMatchSnapshot();
  });
});
