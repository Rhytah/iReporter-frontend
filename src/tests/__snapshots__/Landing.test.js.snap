// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Component matches the snapshot 1`] = `
<div>
  <body
    id="page-top"
  >
    <nav
      id="navigation"
    >
      <div
        className="container text-center"
      >
        <ul
          className="navHold"
        >
          <li>
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Redflags
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Interventions
            </a>
          </li>
          <li
            className="subNav"
          >
            Get started
            <ul>
              <li>
                <a
                  href="/"
                >
                  Sign up
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Login
                </a>
              </li>
            </ul>
          </li>
        </ul>
      </div>
      <div
        className="logosmall hidels"
      >
        <img
          alt="pic"
          height="24px"
          src="assets/img/testlogosmall.png"
        />
      </div>
      <div
        className="navResponsive"
      />
    </nav>
    <div
      className="container"
    >
      <article>
        <h1>
          Welcome to iReporter
        </h1>
        <p>
          Corruption is a huge bane to Africaâ€™s development. African countries must develop novel and localised solutions that will curb this menace, hence the birth of iReporter. iReporter enables any/every citizen to bring any form of corruption to the notice of appropriate authorities and the general public. Users can also report on things that needs government intervention
        </p>
      </article>
    </div>
  </body>
</div>
`;
